src/main/java/me/zeroeightsix/kami/util/Coord.java

```diff
@@ -0,0 +1,38 @@
+package me.zeroeightsix.kami.util;
+
+import net.minecraft.util.math.BlockPos;
+
+/**
+ * @author wnuke
+ * Created by wnuke on 17/04/20
+ */
+
+public class Coord {
```

Comment:
This is a misleading classname, perhaps change to `CoordinateLog` or something similar? A "Coord" should not hold date, time, and name information.

Reply:
Makes sense

Reply:
This is now fixed.